mission_coordinator:
  waypoints_x: [ -30, 30] #[m]
  waypoints_y: [ 20, 20] #[m]
  bridges: [0, 0]

  waypoint_error:
    maximum_accepted_waypoint_error: 5
    minimum_waypoint_error: 1
    decrease_waypoint_error: 1

  obstacles:
    start_x:        [20] #[m]
    start_y:        [17] #[m]
    start_theta:    [3.14] #[rad]
    u:              [0.5] #[m/s]
    v:              [0] #[m/s]
    omega:          [0] #[rad/s]
    major_semiaxis: [5] #[m]
    minor_semiaxis: [5] #[m]

  reference:
    velocity: 3.0

  number_of_fuzzy_waypoints: 0
  use_fuzzy: false
  use_PID: false
  safety_margin: 3.0
  frame_id: '/my_frame'
  angle_offset: -1.3
  verbose: true


Local_planner:
  step_size: 1
  weighted_map_border: 5
  use_obstacles: true
  use_extended_obstacles: true
